src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -490,6 +492,20 @@ struct Query {
   7: optional FieldQuery       field
 }
 
+struct AggregateTilesRequest {
+	1: required string sourceNameSpace
+	2: required string targetNameSpace
+	3: required i64 rangeStart
+	4: required i64 rangeEnd
+	5: required string step
```

Comment:
Probably best to make this i64 with a "optional TimeType stepType = TimeType.UNIX_MILLISECONDS" to match other params in the API.

Then you can use:
```go
	step, err := convert.ToDuration(req.StepType)
```

Reply:
As done with things that are durations in the RPC already, see here:
https://github.com/m3db/m3/blob/09ff0df9650d563ea5670e6bd7d3ff369c87477f/src/dbnode/network/server/tchannelthrift/node/service.go#L1933

Reply:
Ah, we are only using this endpoint manually, where it is more convenient (and less error prone) to vary the step like `10m`, '1h`, etc. And this parameter is supposed to go away anyways, as the step should become a part of Computed Namespace metadata.
Let me know what you think.

Reply:
If this is mainly used for by hand then yes, makes sense.